### DISCUSSION TITLE
Startup sequence

### DISCUSSION THREAD

**Discussion by paulmelis** [StereoKit/StereoKit#328] [Q&A] [answered]
One of the first things one of my apps does is make a network connection, pretty much right before entering the main `while (SK.Step())` loop. Monitoring the backend I see that the connection is made very quickly after starting the application from the main menu, but then it takes a couple of seconds to go through the usual "blue wavy blocks transition with sound" bit before the UI becomes visible. I assume the latter part of the startup sequence is being forced by the HoloLens Windows OS itself? I mean, there's no setting to disable that part is there, as it would shave a few seconds off from the startup, which is handy when doing lots of testing iterations ;-)

**Comment by maluoi** [comment, marked as answer]
Yeah, StereoKit starts up _really_ fast! The transition effect is part of the OS, and I don't really know of a way to circumvent it. It's funny how stuff like that can become the bottleneck once you've fixed everything else! 😅
